Automattic/sensei
# Change learner management actions

Fixes https://github.com/Automattic/sensei-wc-paid-courses/issues/497

This PR is a sequence of https://github.com/Automattic/sensei/pull/3404

### Changes proposed in this Pull Request

* This PR introduces the actions to the learner management screen following these requisites: https://github.com/Automattic/sensei-wc-paid-courses/issues/497
* Some errors will occur in this code until the #3402 be merged because some of the methods used here are created there.
* After it merged, the feature still won't work properly until `Sensei_Course_Enrolment::is_enrolled` supports the new feature to remove users.
  * There are some tests already written failing on purpose. This should stop fails when the `Sensei_Course_Enrolment::is_enrolled` supports the new feature to remove users.

### Testing instructions

For now, it's possible only test if the screen is showing the correct buttons in the Learner Management.

If you don't have:
* Create a course.
* Create some users.
* Start the course progress with some users.
* Manually enroll some users (Enroll link) and unenroll others.
* Enroll a user with another provider.

Then:
* Go to WP admin > Sensei LMS > Learner Management.
* Choose a course to manage the learners.
* Check if it's matching the requisites from the issue: https://github.com/Automattic/sensei-wc-paid-courses/issues/497
